### PR TITLE
Svyat/feat/book creating

### PR DESCRIPTION
**Pull request by SvyatoslavArtymovych**

### CODE REVIEW

File: app/forms/book.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField, SelectField
+from wtforms.validators import DataRequired, Length
+
+from app import models as m
+
+
+class CreateBookForm(FlaskForm):
+    label = StringField("Label", [DataRequired(), Length(6, 1024)])
```
Comment (kipod) [inline app/forms/book.py:9]: IMHO: Length(6, 1024) -> Length(1, 128)

File: app/views/book.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    else:
+        log(log.ERROR, "Book create errors: [%s]", form.errors)
+        for field, errors in form.errors.items():
+            field_label = form._fields[field].label.text
+            for error in errors:
+                flash(error.replace("Field", field_label), "danger")
+        return redirect(url_for("book.get_all"))
+
+
+@bp.route("/<int:book_id>/settings", methods=["GET", "POST"])
```
Comment (kipod) [inline app/views/book.py]: POST - may be deleted

File: app/views/book.py [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        for field, errors in form.errors.items():
+            field_label = form._fields[field].label.text
+            for error in errors:
+                flash(error.replace("Field", field_label), "danger")
+        return redirect(url_for("book.settings", book_id=book_id))
+
+
+@bp.route("/<int:book_id>/edit_contributor_role", methods=["POST"])
+@login_required
+def edit_contributor_role(book_id):
```
Comment (kipod) [inline app/views/book.py]: book_id: int

**Review by kipod — APPROVED** [review]
Thanks

File: app/forms/book.py (hunk 1 above)
Comment (SvyatoslavArtymovych) [reply app/forms/book.py:9]: Lets ask Monica about label lengths

**Pull request merged**